astroquery/query.py

```diff
@@ -205,25 +205,28 @@ def _request(self, method, url, params=None, data=None, headers=None,
             a list of strings containing the downloaded local paths if ``save``
             is True
         """
+        req_kwargs = dict(
+            params=params,
+            data=data,
+            headers=headers,
+            files=files,
+            timeout=timeout
+        )
         if save:
             local_filename = url.split('/')[-1]
             if os.name == 'nt':
                 # Windows doesn't allow special characters in filenames like
                 # ":" so replace them with an underscore
                 local_filename = local_filename.replace(':', '_')
-            local_filepath = os.path.join(self.cache_location or savedir or
-                                          '.', local_filename)
+            local_filepath = os.path.join(self.cache_location or savedir or '.', local_filename)
             # REDUNDANT: spinner has this log.info("Downloading
             # {0}...".format(local_filename))
-            self._download_file(url, local_filepath, timeout=timeout,
-                                auth=auth, cache=cache,
-                                continuation=continuation)
+            self._download_file(url, local_filepath, cache=cache, continuation=continuation, method=method, auth=auth,
+                                **req_kwargs)
```

Comment:
actually I prefer it to keep passing timeout directly rather than the full `req_kwargs` dict, passing arguments the function wont use may lead to issues in the future? I'm not sure, so this is also not a blocker.